src/pages/audio-rewards-page/store/sagas.ts

```diff
@@ -176,19 +178,34 @@ function* claimChallengeRewardAsync(
             yield put(
               setVisibility({ modal: HCAPTCHA_MODAL_NAME, visible: true })
             )
+            yield put(claimChallengeRewardWaitForRetry(claim))
             break
           case FailureReason.COGNITO_FLOW:
             yield put(
               setVisibility({ modal: COGNITO_MODAL_NAME, visible: true })
             )
+            yield put(claimChallengeRewardWaitForRetry(claim))
+            break
+          case FailureReason.ALREADY_DISBURSED:
+            yield put(claimChallengeRewardAlreadyClaimed())
+            break
+          case FailureReason.ALREADY_SENT:
+            if (retryOnFailure) {
+              // In this case, we should retry, but give a bit of time.
+              yield delay(2000)
+              yield put(claimChallengeReward({ claim, retryOnFailure: false }))
```

Comment:
Don't think we should retry here - this is error means rewards manager has already created the account representing the transfer, so it's never going to succeed